ccip/config/evm/Monad_Testnet.toml

```diff
@@ -0,0 +1,23 @@
+ChainID = '10143'
+# finality_depth was: 0
+FinalityDepth = 10
```

Comment:
No finality tag? & did the finality committee approve the number?

Reply:
I haven't reached out, was going off a previous conversation I had with @fernandezlautaro  regarding finality. Where a chain with instant finality defaults to a finality depth of 10 (example being sei). Please let me know if I should

Reply:
Yes please atleast make them aware, Max would be a good person to start with